prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -1917,6 +1929,11 @@ struct rxr_rx_entry *rxr_pkt_alloc_rta_rx_entry(struct rxr_ep *ep, struct rxr_pk
 		return NULL;
 	}
 
+	if (op == ofi_op_atomic) {
```

Comment:
Not sure I follow this change?

Reply:
In master branch, then receiver calls `rxr_pkt_alloc_rta_rx_entry` only for ofi_op_atomic_fetch operation and ofi_op_atomic_compare operation. However, in this pull request. `rxr_pkt_alloc_rta_rx_entry` is also called by the receiver for `ofi_op_atomic ` operation. In this case we should not touch the resting logic for ofi_op_atomic_fetch operation and ofi_op_atomic_compare operation.